examples/python-docker/README.org

```diff
@@ -0,0 +1,87 @@
+#+title: Python-Docker
```

Comment:
I would use the [markdown](https://en.wikipedia.org/wiki/Markdown) format for the README.

Reply:
Also, please follow the guide shown [here](https://github.com/hashicorp/terraform-cdk/tree/master/examples/python-aws#python-aws) and update the README with the instructions accordingly.

Reply:
README updated as per the guide and formatting changed to markdown.